### ISSUE TITLE
`jsdoc/check-tag-names` conflicts with `unicorn/isolated-functions`

### ISSUE THREAD

**Issue by fregante** [xojs/eslint-config-xo#116]
`unicorn/isolated-functions` uses `@isolated` but `jsdoc/check-tag-names` doesn't allow it:

```js
/** @isolated */
function doStuff(){}
```

`jsdoc/require-description` is also annoying here but alas I don’t think specific tags can be excluded

**Comment by fregante** [comment]
Strangely I'm still getting both rules reporting on here

```ts
export const defaultBranchOfRepo = new CachedFunction('default-branch', {
	/** @isolated */
	// eslint-disable-next-line jsdoc/require-description, jsdoc/check-tag-names
	async updater(repository: NameWithOwner): Promise<string> {
		if (!repository) {
			throw new Error('getDefaultBranch was called on a non-repository page');
		}


		return (isCurrentRepo(repository) && await fromDom()) || fromApi(repository);
	},

	maxAge: {hours: 1},
	staleWhileRevalidate: {days: 20},
});
```
```
eslint-config-xo@0.57.0
  └── eslint-plugin-unicorn@71.1.0
```

`unicorn/isolated-functions` is running correctly on the method though (https://github.com/sindresorhus/eslint-plugin-unicorn/issues/3463) so I'm definitely on v70+

**Comment by sindresorhus** [comment]
The `@isolated` handling works correctly here since 0.56.0, and I can't reproduce any `jsdoc/check-tag-names` or `jsdoc/require-description` error on your snippet with `eslint-config-xo@0.57.0` + `eslint-plugin-jsdoc@63.0.12`.

What you're actually seeing is the opposite of what it looks like. Both rules are now silent on that method, which makes your `// eslint-disable-next-line jsdoc/require-description, jsdoc/check-tag-names` an unused directive. ESLint reports that as:

> Unused eslint-disable directive (no problems were reported from 'jsdoc/require-description' or 'jsdoc/check-tag-names').

That message names both rules, so it reads like "both rules still fire", but it means the disable is no longer needed.

Just delete the `eslint-disable-next-line` comment and it'll be clean.

**Comment by fregante** [comment]
It was something sillier: vscode did not load the new eslint plugins :(
